### PR TITLE
Add Gradle to akka/java sample

### PR DESCRIPTION
**Pull request by marcospereira**
This can be later added to Akka Platform Guide, but let's start here. The idea is that we can link this in our docs for Gradle.﻿

### CODE REVIEW

File: akka/shopping-cart-java/build.gradle [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    }
+}
+
+dockerCreateDockerfile.configure { finalizedBy copyAgentJar }
+
+docker {
+    javaApplication {
+        baseImage = "docker.io/library/adoptopenjdk:11-jre-hotspot"
+        mainClassName = "shopping.cart.Main"
+        jvmArgs = ["-javaagent:/libs/cinnamon-agent.jar"]
```
Comment (ygree) [inline akka/shopping-cart-java/build.gradle:119]: Should it be `-javaagent:${configurations.agent.singleFile}` instead?

Comment (marcospereira) [reply akka/shopping-cart-java/build.gradle:119]: No, this is for the generated docker image. All the files, including the agent, are `COPY`ed to `/libs` inside the image. Some customization was necessary, though: https://github.com/lightbend/telemetry-samples/pull/31/files#diff-4a38a3bf4e4b3450faf1f5caa9af93f03bf57502dc8e9fbc0bfb6439d0dc39dcR103-R111

Comment (ygree) [reply akka/shopping-cart-java/build.gradle:119]: Got it. Thanks for the explanation!

**Review by ygree — APPROVED** [review]
LGTM

**Pull request merged**